tests/components/bitvavo/test_config_flow.py

```diff
@@ -0,0 +1,187 @@
+"""Test the Bitvavo config flow."""
+import json
+from unittest.mock import patch
+
+from bitvavo.BitvavoExceptions import BitvavoException
+
+from homeassistant import data_entry_flow, setup
+from homeassistant.components.bitvavo.const import (
+    CONF_API_SECRET,
+    CONF_MARKETS,
+    CONF_SHOW_EMPTY_ASSETS,
+    DOMAIN,
+)
+from homeassistant.config_entries import SOURCE_USER
+from homeassistant.const import CONF_API_KEY, CONF_SOURCE
+
+from . import INTEGRATION_TITLE, USER_INPUT, USER_INPUT_MARKETS
+
+from tests.common import MockConfigEntry, load_fixture
+
+
+async def test_show_form(hass):
+    """Test that the form is served with no input."""
+    await setup.async_setup_component(hass, "persistent_notification", {})
```

Comment:
You don't appear to use persistent notifications anywhere in config flow, so why are you adding it to the tests?

Reply:
Copy/paste error, fixed.